application/src/main/data/upgrade/3.6.4/schema_update.sql

```diff
@@ -134,3 +134,33 @@ DELETE FROM asset WHERE type='TbServiceQueue';
 DELETE FROM asset_profile WHERE name ='TbServiceQueue';
 
 -- QUEUE STATS UPDATE END
+-- TENANT PROFILE UPDATE START
+
+UPDATE tenant_profile tp
```

Comment:
please rewrite this to 3 separate statements (one per configuration item, e.g. transportDeviceMsgRateLimit ) and also check that the field you are using is set. For example, that the transportDeviceMsgRateLimit is set.